<launch>
  <arg name="gui" default="true" />
  <arg name="model" default="deeplab_v3plus_xception65"/>
  <arg name="pretrained_model" default="ade20k"/>
  <arg name="gpu" default="-1"/>
  <arg name="label_names"
       default="$(find ros_chainercv)/yaml/ade20k_semantic_segmentation_label_names.yaml" />

  <include file="$(find ros_chainercv)/sample/sample_semantic_segmentation.launch">
    <arg name="gui" value="$(arg gui)" />
    <arg name="model" value="$(arg model)" />
    <arg name="pretrained_model" value="$(arg pretrained_model)" />
    <arg name="gpu" value="$(arg gpu)" />
    <arg name="label_names" value="$(arg label_names)" />
  </include>

</launch>
